package/kedro_viz/server.py

```diff
@@ -351,7 +351,12 @@ def format_pipelines_data(pipelines: Dict[str, "Pipeline"]) -> Dict[str, list]:
     }
 
 
-# pylint: disable=too-many-locals,too-many-arguments
+def _is_namespace_param(namespace: str) -> bool:
+    """Returns whether a dataset namespace is a parameter"""
+    return "param" in namespace.lower()
```

Comment:
I know this is what it was doing before (line 440), but I think technically `namespace.lower().startswith("param")` is more correct.